src/components/TaskRunnerPaneRow/TaskRunnerPaneRow.js

```diff
@@ -52,9 +52,13 @@ class TaskRunnerPaneRow extends PureComponent<Props> {
         </StatusColumn>
 
         <LinkColumn>
-          <StrokeButton size="small" onClick={() => onViewDetails(name)}>
-            View Details
-          </StrokeButton>
+          {name !== 'eject' ? (
+            <StrokeButton size="small" onClick={() => onViewDetails(name)}>
+              View Details
+            </StrokeButton>
+          ) : (
+            ''
```

Comment:
Nit: I think `null` is a slightly clearer value for this than an empty string, and still perfectly valid JSX

Also, how do you feel about reordering them? My rationale is that it's easier to follow when the truthy case is first and the falsy case is second; there's almost like a double-negative kind of thing when you're checking for not-equals instead of equals.

Together, this is what I imagine:

```js
{name === 'eject' ? (
  null
) : (
  <StrokeButton size="small" onClick={() => onViewDetails(name)}>
    View Details
  </StrokeButton>
)
```